docs/installation/migrate.rst

```diff
@@ -0,0 +1,90 @@
+.. _schemaregistry_migrate:
+
+Migrate an Existing |sr| to |ccloud| 
+====================================
+
+:ref:`cloud-home` is a fully managed streaming data service based on open source
+|ak-tm|. Just as you can “lift and shift” your Kafka applications from
+self-managed Kafka to Confluent Cloud, you can do the same with |sr-long|.
+
+If you already use |sr| to manage schemas for Kafka applications, and want
+to move some or all of that data and schema management to the cloud, you can use
+:ref:`connect_replicator` to migrate your existing schemas to |sr-ccloud|.
+
+Both bridge to cloud use cases ("lift and shift" all to cloud and "extend to
```

Comment:
@londoncalling Let's change this part to include both one-time and continuous migration.

Continuous migration: You can use your self-managed Schema Registry as a primary and Confluent Cloud Schema Registry as a secondary. New schemas will be registered directly to the self-managed Schema Registry, and Replicator will continuously copy schemas from it to the Confluent Cloud Schema Registry, which should be set to IMPORT mode.

One-time migration: This is great if you intend to migrate your existing self-managed Schema Registry to Confluent Cloud Schema Registry as a primary. All new schemas are registered to Confluent Cloud Schema Registry. In the scenario, there is no migration from Confluent Cloud Schema Registry back to the self-managed Schema Registry.